prov/verbs/src/verbs_info.c

```diff
@@ -154,8 +154,7 @@ const struct verbs_ep_domain verbs_dgram_domain = {
 	.caps			= VERBS_DGRAM_CAPS,
 };
 
-struct fi_ibv_rdm_sysaddr
-{
+struct fi_ibv_rdm_sysaddr {
```

Comment:
Please drop this change or move it to a separate patch.

Reply:
Dropped

Reply:
It seems the change hasn't been dropped.

Reply:
Fixed